menas/src/main/scala/za/co/absa/enceladus/menas/controllers/PropertyDefinitionController.scala

```diff
@@ -75,4 +86,33 @@ class PropertyDefinitionController @Autowired()(propertyDefService: PropertyDefi
     super.getVersionDetail(propertyName, version)
   }
 
+  @PostMapping(Array("/importItem"))
```

Comment:
New endpoint?

Reply:
Parent endpoints. Not new, they just check for the switch and then call super